src/bin/download_sysext.rs

```diff
@@ -342,7 +340,7 @@ async fn main() -> Result<(), Box<dyn Error>> {
     fs::create_dir_all(&temp_dir)?;
 
     // The default policy of reqwest Client supports max 10 attempts on HTTP redirect.
-    let client = Client::builder().redirect(Policy::default()).build()?;
+    let client = Client::builder().connect_timeout(Duration::from_secs(HTTP_CONN_TIMEOUT)).redirect(Policy::default()).build()?;
```

Comment:
```suggestion
    let client = Client::builder().connect_timeout(Duration::from_secs(HTTP_CONN_TIMEOUT)).timeout(Duration::from_secs(DOWNLOAD_TIMEOUT)).redirect(Policy::default()).build()?;
```
This got lost in the rework. It's not perfect but better than hanging without end when hitting this case.

Reply:
With something like 3600 seconds